test/TensorFlowRuntime/accelerator_only.swift

```diff
@@ -21,6 +21,27 @@ func matmul_and_add(_ a : Tensor<Float>, _ b : Tensor<Float>) -> Tensor<Float> {
   return add_constant(a • b)
 }
 
+// CHECK-LABEL: @{{.*}}emptyArrayAttributes{{.*}}
+// CHECK: graph_op "s16accelerator_only20emptyArrayAttributesy10TensorFlow{{.*}}
+// CHECK-NEXT: return
+// CHECK-NEXT: end sil function {{.*}}emptyArrayAttributes{{.*}}
+@TensorFlowGraph
+@inline(never)
+func emptyArrayAttributes(_ v: StringTensor) -> TensorHandle<Int64> {
```

Comment:
Maybe we can test against the simplified reproducer here?

Reply:
Are you suggesting the one from the bug? Since I want to verify that graph extraction happens, I think I need to test against an actual op? #tfop("Foo", ...) would result in a compile error.

Reply:
Since this is a DA/constexpr bug, i think it's better to add it to a DA test file instead of a partitioning test file.

Reply:
LGTM